integrations/electricity/__tests__/energinet.test.js

```diff
@@ -0,0 +1,448 @@
+import moment from 'moment';
+import energinet from '../energinet';
+
+const fromMoment = moment()
+  .subtract(3, 'days')
+  .startOf('day');
+const toMoment = moment().startOf('hour');
```

Comment:
These constants are only used once, so let's move them down to the test where we need 'em :)